projects/plugins/jetpack/_inc/client/recommendations/sidebar/mobile-app/index.jsx

```diff
@@ -0,0 +1,86 @@
+/**
+ * External dependencies
+ */
+import { createInterpolateElement } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+import React, { useCallback, useEffect } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import AppsBadge from './apps-badge';
+import { Layout } from '../layout';
+import Gridicon from 'components/gridicon';
+import { imagePath } from 'constants/urls';
+import analytics from 'lib/analytics';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+const MobileApp = () => {
+	const features = [
+		__( 'Refined post and page editor', 'jetpack' ),
+		__( 'Manage multiple sites from one dash', 'jetpack' ),
+		__( 'Multi-site plugin management', 'jetpack' ),
+		__( 'Free stock photo library', 'jetpack' ),
+		__( 'Update your site from any device', 'jetpack' ),
```

Comment:
Should we include "Refined post and page editor" here, now that the editor in wpcom is an iFramed version of the wp-admin editor? Same with "Free stock photo library", now that the feature is available in wp-admin?

Reply:
hey hey @jeherve I think we should still include those list items. Even if they're available in wp-admin, many people probably don't know about them, and they're still good marketing tidbits for the apps I think.